<?xml version="1.0"?>
<launch>

  <include file="$(find pal_wbc_tutorial)/launch/upload_rrbot.launch" />

  <!-- Configuring Controllers -->
  <include file="$(find pal_wbc_tutorial)/launch/controllers_configuration.launch"/>

  <!-- Pushing the reference task -->
  <node name="push_reference_pos_task_topic" pkg="pal_wbc_tutorial" type="push_reference_pos_task_topic" />

  <!-- Show in Rviz -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find pal_wbc_tutorial)/launch/rrbot_rviz.rviz"/>

</launch>

